src/main/java/ch/nine/confluence/confidentiality/api/model/Confidentiality.kt

```diff
@@ -0,0 +1,35 @@
+package ch.nine.confluence.confidentiality.api.model
+
+import javax.xml.bind.annotation.XmlElement
+import javax.xml.bind.annotation.XmlRootElement
+
+/**
+ * Simple transport object, that holds information about confidentiality
+ * for JavaScript part of the plugin.
+ */
+@XmlRootElement
+class Confidentiality(private val enabled: Boolean,
+                      private val confidentiality: String?,
+                      private val possibleConfidentialities: List<String>?,
+                      private val canUserEdit: Boolean?) {
+    @XmlElement
+    fun isEnabled() : Boolean {
```

Comment:
I know it's not that important. My preference in thos cases is to write the minimal version. Types should be implied automatically by the enabled type aswell.
```kotlin
fun isEnabled() = enabled
```